contrib/tag-release.py

```diff
@@ -1,32 +1,16 @@
 def main():
-    with open('src/app/fdctl/version.mk', 'r') as f:
-        lines = f.readlines()
-
-    version_major = None
-    version_minor = None
-    version_patch = None
-    for line in lines:
-        if line.startswith('VERSION_MAJOR'):
-            version_major = int(line.split(':=')[1].strip())
-        elif line.startswith('VERSION_MINOR'):
-            version_minor = int(line.split(':=')[1].strip())
-        elif line.startswith('VERSION_PATCH'):
-            version_patch = int(line.split(':=')[1].strip())
-        else:
-            print('Error: version.mk file is not well formatted')
-            exit(1)
-
-    if version_major is None or version_minor is None or version_patch is None:
-        print('Error: version.mk file is not well formatted')
-        exit(1)
+    version_txt = open('src/app/fdctl/version.txt', 'r').read().strip()
+    version_parts = version_txt.split('.')
+    version_major = int(version_parts[0])
```

Comment:
`version_major` is always 0, the version in this file is actually <minor>.<patch>